docs/src/main/asciidoc/hibernate-orm-panache.adoc

```diff
@@ -618,6 +618,15 @@ so the compiler must be configured to store parameter names inside the compiled
 This is enabled by default if you are using the Quarkus Maven archetype. If you are not using it, add the property `<maven.compiler.parameters>true</maven.compiler.parameters>` to your pom.xml.
 ====
 
+== Multiple Persistence Units
+
+The support for multiple persistence units is described in details in link:hibernate-orm#multiple-persistence-units[the Hibernate ORM guide].
+
+When using Panache, things are simple:
+
+* A given Panache entity can be attached to only one persistence unit;
+* Given that, things are totally transparent and Panache automatically uses the appropriate `EntityManager`.
```

Comment:
We should refrain from calling "things" since it's too subjective. Perhaps remove this part since it's explained in the linked document

Reply:
Or put something like this before the initial sentence containing the link to the guide: 

```
A given Panache entity can be attached to only one persistence unit. Given that, Panache already provides the necessary plumbing to find the appropriate `EntityManager` associated to a Panache entity
```

Reply:
Okay. I don't see the problem with `things` but changed it to your version.

Reply:
Thanks , but my point was that `things` is also used in the `things are simple` and that confuses the reader as to what `things` is referring to. Also simple != easy,  that's why it confused be a bit